PF/Assignment3/Reminder.java

```diff
@@ -0,0 +1,19 @@
+public class Reminder {
+
+	public static int rem(int x, int y) {
+		if (x >= 0 && y > 0) {
+
+			if (x >= y) {
+				return rem(x - y, y);
+			} else
+				return x;
+		} else {
+			System.out.print("Divide by zero not possible");
+
+			return -1;
```

Comment:
Yeah, That is why I had already removed the comment :) But the print message is still misleading.